PasswordVerifier/src/main.java

```diff
@@ -50,9 +57,9 @@ public static boolean correctnessTest() {
      * stress tests isValid function. prints time to check test cases
      * @param count number of test cases to run
      */
-    public static void stressTest(int count) {
-        int minLength = 5;
-        int maxLength = 20;
+    public static void stressTest(int count, boolean logging) {
```

Comment:
logging is passed as a parameter rather than resolved at the time of log entry because of the time consideration? If so, a comment would help